src/CommandList.java

```diff
@@ -0,0 +1,36 @@
+
+import java.util.HashMap;
+import java.util.Map;
+public class CommandList {
+    HashMap<String,Command> testCommand;
+    private String add = "add";
```

Comment:
Looks like constant field. Check how constant fields should be decelerated : https://java-blog.ru/osnovy/konstanta-java